modules/ROOT/pages/mmp-concept.adoc

```diff
@@ -794,22 +833,20 @@ Optional parameters:
 
 === Deploy Goal
 
-Uploads and deploys the application JAR file to any of the application deployment targets.
-Bind's by default to the http://maven.apache.org/ref/3.6.1/maven-core/lifecycles.html[lifecycle phase]: deploy
-
-Optional parameters:
+This goal uploads and deploys the application JAR file to any of the application deployment targets.
+Bind's by default to the http://maven.apache.org/ref/3.6.1/maven-core/lifecycles.html[lifecycle phase]: `deploy`
 
+.Optional parameters
 [%header%autowidth.spread,cols="a,a,a"]
 |===
 |Parameter | Type | Description
 |`artifact` | String | Path to the application JAR file to be deployed. By default is set to the application `target` folder.
-|`muleDeploy` | boolean | If it is set to `true`, the plugin performs a deploy using the deployment strategy defined in the plugin configuration. +
-If it is set to `false`, it uploads the artifacts to the repository defined in the `distributionManagement` section on the application's `pom.xml` file.
+|`muleDeploy` | boolean | Instructs the plugin to deploy using the deployment strategy defined in the plugin configuration. If it is not set, the plugin uploads the artifacts to the repository defined in the `distributionManagement` section on the application's `pom.xml` file.
```

Comment:
```suggestion
|`muleDeploy` | boolean | Instructs the plug-in to deploy using the deployment strategy defined in the plug-in configuration. If it is not set, the plug-in uploads the artifacts to the repository defined in the `distributionManagement` section of the application's `pom.xml` file.
```
QUESTION: Does "if it is not set" refer to this parameter not being set or to the deployment strategy not being defined in the plug-in?